src/data/finsupp/basic.lean

```diff
@@ -1134,10 +1141,24 @@ lemma update_eq_sub_add_single [add_group G] (f : α →₀ G) (a : α) (b : G)
   f.update a b = f - single a (f a) + single a b :=
 by rw [update_eq_erase_add_single, erase_eq_sub_single]
 
+/-- Given a family of functions `f i : α → N` indexed over `S : finset ι`, the sum of this family
+  over `S` is a function `α → N` whose value at `p : α` is `∑ (i : ι) in S, (f i) p` -/
```

Comment:
This comment is wrong (it's about `→` not `→₀`). I also think it's unnecessary, and is harder to read than the lemma statement itself:

```suggestion
```